src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
-body {
+$accent-color: #060b35;
+$secondary-color: #616070;
+$button-color: #00acdc;
+
+* {
   margin: 0;
```

Comment:
don't reset margin in `*` selector
reset it for the list of properties you really need